lib/ViewModels/DataCatalogTabViewModel.js

```diff
@@ -73,12 +73,12 @@ DataCatalogTabViewModel.prototype.getCheckboxClass = function(item) {
 	return 'data-catalog-item-checkbox unchecked';
 };
 
-DataCatalogTabViewModel.prototype.getRightSideItemControls = function(item) {
-    return CatalogItemControl.rightSideItemControls(item);
+DataCatalogTabViewModel.prototype.getRightSideMemberControls = function(catalogMember) {
```

Comment:
Provide the old name, too (with a `deprecationWarning`) for backward compatibility here too.